packages/api/es/esScrollSearch.js

```diff
@@ -2,7 +2,7 @@
 
 const { Search } = require('./search');
 const defaultESScrollSize = 1000;
-const defaultESScrollDuration = '30s';
+const defaultESScrollDuration = '2m';
```

Comment:
This was the main issue is that the search context was dying. I did some reading and increased it. I exposed it an env var (to just us for now) in case we need to test with increasing it or decreasing it on the fly.